opacus/layers/dp_lstm.py

```diff
@@ -49,17 +93,31 @@ def reset_parameters(self):
             nn.init.uniform_(weight, -stdv, stdv)
 
     def forward(
-        self, x: torch.Tensor, h_prev: torch.Tensor, c_prev: torch.Tensor
+        self,
+        x: torch.Tensor,
+        h_prev: torch.Tensor,
+        c_prev: torch.Tensor,
+        B: Optional[int] = None,
```

Comment:
Who's B? What does it do?

Reply:
Since ```B``` was used to denote the batch size, I used it here. This variable represents the batch size at time step t. I will rename it to ```batch_size_t```. When PackedSequence is not passed into the DPLSTM, this variable will be ```None``` and so the ```forward``` function will operate like before. Since the rows of ```h_prev``` and ```c_prev``` are sorted based on the sequence lengths, ```B``` is used to select the values in h_prev and c_prev corresponding to only the top ```B``` sequences.